junit-platform-console/src/main/java/org/junit/platform/console/options/AvailableOptions.java

```diff
@@ -202,6 +203,9 @@
 			"Provide the ID of an engine to be excluded from the test run. This option can be repeated.") //
 				.withRequiredArg();
 
+		failIfNoTests = parser.accepts("fail-if-no-tests",
+			"Fail(return error exit code) if no tests are found to be executed.");
+
```

Comment:
Since this is not a _filter_, please move it to the bottom of the "General Purpose" section above.